packages/flutter/test/material/theme_test.dart

```diff
@@ -254,13 +255,13 @@ void main() {
     );
     await tester.pump(const Duration(milliseconds: 100)); // Halfway through the theme transition
 
-    glyphText = tester.renderObject(find.byType(RichText));
+    glyphText = tester.renderObject(find.byType(RichText, skipOffstage: false));
```

Comment:
> ExcludeSemantics shouldn't mean something is offstage.

As I mention above it depends on how we want to look at semantics. It might make sense to take semantics into account by default in tests, because the vast majority of tests interacting with the widget would treat is as a blackbox. Case in point, I am testing a `MyForm` widget, which embeds a `DropdownButton`. My test might want to interact with the `MyForm` widget in a white box fashion and probe into its internals, however, it probably wants to interact with `DropdownButton` according to its external semantics.